### PR TITLE
SR2019 Registration and Kickstart Announcement

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Add announcement email for SR2019.

## Details to fill
- Signup link: This will be a link to the new MailChimp list for SR2019
- Contact email: This will be teams@sr.

### CODE REVIEW

**Comment by Adimote** [comment]
Make sure the [I'd like to submit a team'] is a really big and obvious button!

File: SR2019/2018-09-27-kickstart-announcement.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+to: All Teams
+subject: SR2019 Registration and Kickstart Announcement
+---
+
+After a year's hiatus during which we've worked to ensure that Student Robotics will be more scalable for the future, we're pleased to announce the continuation of the annual competition. As always, we start this competition period with a Kickstart event. Here, we will announce the game, and show you how to use the kit.
+
+In order to allow the core team of volunteers to focus on delivering a high quality programme while requiring far fewer resources than before, this year's competition will be slightly smaller than previous years. We expect to be able to increase the size of the competition again in the future.
+
+Kickstart will be on **Saturday 10th November**. This time, we're planning on running 2 events. 1 at the **University of Southampton**, and the second in **London**. The exact locations will be confirmed in the next couple of weeks, however the events will run from around 10am, and finish some time before 5pm. This information will be reflected on our website soon. If you are unable to make either of these, please let us know!
```
Comment (Adimote) [inline SR2019/2018-09-27-kickstart-announcement.md]: maybe 'at 2 locations' instead of '2 events', it's a bit clearer that it happens at the same time

File: SR2019/2018-09-27-kickstart-announcement.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+If you would like to compete in Student Robotics 2019, please fill in the form at the link below with the required information. All further emails about SR2019 will be sent there. The closing date will be *1st November*.
```
Comment (Adimote) [inline SR2019/2018-09-27-kickstart-announcement.md]: Maybe mention teams will be picked on a first-come first-served basis?

File: SR2019/2018-09-27-kickstart-announcement.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+[I'd like to bring a team to Student Robotics 2019]
```
Comment (Adimote) [inline SR2019/2018-09-27-kickstart-announcement.md]: Do we want to give guidance for schools which wish to submit 2 teams?

File: SR2019/2018-09-27-kickstart-announcement.md (hunk 1 above)
Comment (PeterJCLaw) [inline SR2019/2018-09-27-kickstart-announcement.md]: General comment: I suggest we prefer spelling numbers in words when using them in this sort of context. ("9pm" or "250mm" is fine, but "2 events" reads a little oddly, "1 at .." even more so)

File: package-lock.json [hunk 4]
```diff
@@ -18,23 +18,23 @@
       "resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-3.2.1.tgz",
       "integrity": "sha512-VT0ZI6kZRdTh8YyJw3SMbYm/u+NqfsAxEpWO0Pf9sq8/e94WxxOpPKx9FR1FlyCtOVDNOQ+8ntlqFxiRc+r5qA==",
       "requires": {
-        "color-convert": "^1.9.0"
+        "color-convert": "1.9.3"
```
Comment (PeterJCLaw) [inline package-lock.json]: Maybe cherry-pick these unrelated changes to the `package-lock.json` straight onto `master`?

File: SR2019/2018-09-27-kickstart-announcement.md (hunk 1 above)
Comment (PeterJCLaw) [inline SR2019/2018-09-27-kickstart-announcement.md]: Can we frame this in such a way that it's clear that:
- these events are invite-only, and
- that we'll tell teams which venue to go to

The latter obviously assuming that that is our intent.

File: SR2019/2018-09-27-kickstart-announcement.md (hunk 2 above)
Comment (RealOrangeOne) [reply SR2019/2018-09-27-kickstart-announcement.md]: We never decided this, so let's leave it until we're sure that's how we're going to do it. I'll add something about responding the sooner the better

File: SR2019/2018-09-27-kickstart-announcement.md (hunk 1 above)
Comment (RealOrangeOne) [reply SR2019/2018-09-27-kickstart-announcement.md]: I've added a field to the signup form saying "Desired kickstart location", which I think covers the latter point. 

I think the invite-only component is obvious, seeing as only certain people will be getting all the details anyway.

Comment (PeterJCLaw) [reply SR2019/2018-09-27-kickstart-announcement.md]: ok, for clarity this was particularly in response to:

>  If you are unable to make either of these, please let us know!

which feels like it implies that this email is an invite.

Comment (RealOrangeOne) [reply SR2019/2018-09-27-kickstart-announcement.md]: I'll remove that line, and capture it in the form instead

**Comment by prophile** [comment]
:shipit:

**Comment by RealOrangeOne** [comment]
This email has now shipped!
